src/react-components/home-root.js

```diff
@@ -104,6 +105,13 @@ class HomeRoot extends Component {
     this.showDialog(AuthDialog, { verifying, verified, authOrigin: this.props.authOrigin });
   };
 
+  showHubInfo = hubEntry => {
```

Comment:
I know these terms are getting mashed up, but probably `showRoomInfo` given the dialog name